qlty-cli/src/commands/coverage/complete.rs

```diff
@@ -107,21 +114,36 @@ impl Complete {
         Ok(())
     }
 
-    fn print_complete_success(&self, elapsed_seconds: f32) {
+    fn print_complete_success(&self, elapsed_seconds: f32, url: &str) {
         if self.quiet {
             return;
         }
 
         eprintln!("    Coverage marked as complete in {elapsed_seconds:.2}s!");
+
+        if !url.is_empty() {
+            eprintln!("    {}", style(format!("View report: {url}")).bold());
+        }
+
         eprintln!();
     }
 
     fn request_complete(
         metadata: &qlty_types::tests::v1::CoverageMetadata,
         token: &str,
-    ) -> Result<Value> {
+    ) -> Result<CompleteResult> {
         let legacy_api_url = get_legacy_api_url();
         let client = QltyClient::new(Some(&legacy_api_url), Some(token.into()));
-        client.post_coverage_metadata("/coverage/complete", metadata)
+        let response = client.post_coverage_metadata("/coverage/complete", metadata)?;
+
+        let url = response
+            .get("data")
+            .and_then(|data| data.get("url"))
+            .and_then(|url| url.as_str())
+            .unwrap_or_default();
```

Comment:
Returning an empty string and testing it with `is_empty` is OK but a bit of a fib to the type system since it's a missing value. Slightly better would be to return `Option<String>` and don't print if `None` is returned.